components/Ink/tests/unit/MDCInkLayerTests.m

```diff
@@ -147,16 +147,13 @@ - (void)testEndAnimationTimingInTimeScaledLayer {
 
   // When
   NSTimeInterval startTime = CACurrentMediaTime();
+  startTime = [inkLayer convertTime:(startTime + 0.9) fromLayer:nil];
```

Comment:
I'm probably missing something here.  How does moving the time computation affect the outcome of the test? Unless the receiver's time scale has changed relative to the caller (which I don't believe it does in this case), I fail to see how it can make a difference.

Reply:
There's a difference as the computation went chronologically in between the animation.beginTime and the startTime. However, this will make it even less optimal because it's better to get the computation after both times are received. Updated.

Reply:
It looks like there is less branching if we reorder the beginTime and the startTime, it gives closer accuracy in that case; have switched between the two.